tests/unit_tests/test_serpent_depcode.py

```diff
@@ -6,84 +6,93 @@
 
 
 def test_create_nuclide_name_map_zam_to_serpent(serpent_depcode):
-    serpent_depcode.create_nuclide_name_map_zam_to_serpent()
-    assert serpent_depcode.iso_map[380880] == '38088.09c'
-    assert serpent_depcode.iso_map[962400] == '96240.09c'
-    assert serpent_depcode.iso_map[952421] == '95342.09c'
-    assert serpent_depcode.iso_map[340831] == '340831'
-    assert serpent_depcode.iso_map[300732] == '300732'
-    assert serpent_depcode.iso_map[511262] == '511262'
-    assert serpent_depcode.iso_map[420931] == '420931'
-    assert serpent_depcode.iso_map[410911] == '410911'
-
-
-def test_convert_nuclide_name_serpent_to_zam(serpent_depcode):
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(47310) == 471101
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(95342) == 952421
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(61348) == 611481
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(52327) == 521271
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(1001) == 1001
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(1002) == 1002
-    assert serpent_depcode.convert_nuclide_name_serpent_to_zam(48315) == 481151
+    nuc_code_map = serpent_depcode.map_nuclide_code_zam_to_serpent()
+    assert nuc_code_map[380880] == '38088.09c'
+    assert nuc_code_map[962400] == '96240.09c'
+    assert nuc_code_map[952421] == '95342.09c'
+    assert nuc_code_map[340831] == '340831'
+    assert nuc_code_map[300732] == '300732'
+    assert nuc_code_map[511262] == '511262'
+    assert nuc_code_map[420931] == '420931'
+    assert nuc_code_map[410911] == '410911'
```

Comment:
Just curious. Is there any reason you chose to test these nuclides?

Reply:
These are holdovers from Andrei's tests.